modules/islandora_job_islandora_batch_ingest/islandora_job_islandora_batch_ingest.drush.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Custom Drush commands.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function islandora_job_islandora_batch_ingest_drush_command() {
+  return array(
+    'islandora_job_islandora_batch_ingest' => array(
+      'aliases' => array('ijibi'),
+      'description' => dt('Ingest a batch set with Islandora Jobs.'),
+      'drupal dependencies' => array(
+        'islandora',
+        'islandora_job',
+        'islandora_job_islandora_batch_ingest',
+      ),
+      'options' => array(
+        'set_id' => array(
+          'description' => dt('The batch set ID to process.'),
+          'required' => TRUE,
+        ),
+      ),
+      'examples' => array(
+        'drush -u 1 -v ijibi --set_id=1' => dt('Sends batch set 1 to Islandoa Jobs for ingesting.'),
+      ),
+    ),
+  );
+}
+
+/**
+ * Drush command to add batch set to Gearman.
+ */
+function drush_islandora_job_islandora_batch_ingest() {
+  $set_id = drush_get_option('set_id', 0);
```

Comment:
Should we validate that the set_id exists and is in the a state to be ingested before we queue it or do we not care?

Reply:
Yea. I can add a bit of validation.